### PR TITLE
feat: added faq and press cards

### PR DESCRIPTION
**Pull request by anshul-pinto0410**
Added the Following Cards
-  Press Card
-  FAQ Card

FAQ
Press

### CODE REVIEW

File: src/components/Card/FAQcard.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.lil-faqcard {
+  /*
+  background-color needs to be updated
+  */
+  height: 280px;
+  width: 252px;
```
Comment (pritam248) [inline src/components/Card/FAQcard.css]: responsive ?

File: src/components/Card/FAQcard.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  @apply lil-rounded-xl lil-p-5  lil-cursor-pointer lil-text-center lil-m-2;
+}
+
+.lil-faqcard > .lil-faqcard-faq-wrapper {
+  @apply lil-mr-5 lil-w-full;
+}
+
+.lil-faqcard-faq-wrapper .lil-faqcard-faq {
+  height: 33%;
+  white-space: normal;
```
Comment (pritam248) [inline src/components/Card/FAQcard.css]: use from tailwind utility class

File: src/components/Card/FAQcard.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  @apply lil-rounded-xl lil-p-5  lil-cursor-pointer lil-text-center lil-m-2;
+}
+
+.lil-faqcard > .lil-faqcard-faq-wrapper {
+  @apply lil-mr-5 lil-w-full;
+}
+
+.lil-faqcard-faq-wrapper .lil-faqcard-faq {
+  height: 33%;
```
Comment (pritam248) [inline src/components/Card/FAQcard.css]: can we use flex here and provide width based on utility exposed by tailwind

File: src/components/Card/FAQcard.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  white-space: normal;
+  @apply lil-text-s-20  lil-text-left lil-text-white lil-font-semibold lil-mr-5;
+}
+
+.lil-faqcard > .lil-faqcard-content {
+  height: 66%;
+  opacity: 0.6;
+  white-space: normal;
```
Comment (pritam248) [inline src/components/Card/FAQcard.css]: SAA

File: src/components/Card/presscard.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
+.lil-presscard {
+  height: 280px;
+  width: 280px;
```
Comment (pritam248) [inline src/components/Card/presscard.css:3]: SAA

File: src/components/Card/presscard.css [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  @apply lil-rounded-xl lil-bg-white lil-align-middle lil-m-4;
+}
+
+/*
+Colors have to be modiefied for .lil-presscard-year & .lil-presscard-award
+*/
+.lil-presscard > .lil-presscard-year {
```
Comment (pritam248) [inline src/components/Card/presscard.css]:
```suggestion
.lil-presscard > .lil-presscard-heading {
```

File: src/components/Card/presscard.css [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  @apply lil-text-base lil-text-black lil-text-center lil-mx-2 lil-my-3 lil-h-12;
+}
+.lil-presscard > .lil-presscard-award {
```
Comment (pritam248) [inline src/components/Card/presscard.css]:
```suggestion
.lil-presscard > .lil-presscard-content {
```

File: tailwind.config.js [hunk 8]
```diff
@@ -79,6 +79,7 @@ module.exports = {
     fontWeight: {
       light: 300,
       normal: 400,
+      semibold: 600,
```
Comment (pritam248) [inline tailwind.config.js:82]: 👍

File: src/components/Card/FAQcard.css (hunk 1 above)
Comment (anshul-pinto0410) [reply src/components/Card/FAQcard.css]: Yes

File: src/components/Card/FAQcard.css (hunk 3 above)
Comment (anshul-pinto0410) [reply src/components/Card/FAQcard.css]: used justify-between along with some margins

**Comment by anshul-pinto0410** [comment]
Added Reinforcement card
reinforce

File: src/components/Card/FAQcard.css (hunk 1 above)
Comment (pritam248) [reply src/components/Card/FAQcard.css]: lets make width `250px` then...we shud not use random values

Comment (anshul-pinto0410) [reply src/components/Card/FAQcard.css]: okay will make it 250, I used the values given in figma

File: src/components/Reinforcement/reinforcement.css [hunk 9]
```diff
@@ -0,0 +1,34 @@
+.lil-reinforcement {
+  height: 408px;
```
Comment (pritam248) [inline src/components/Reinforcement/reinforcement.css:2]: too large for smaller screen...let's try to make it responsive

File: src/components/Reinforcement/reinforcement.css [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+  @apply lil-text-s-48 lil-text-left lil-font-bold;
+}
+
+@screen sm {
+  .lil-reinforcement {
+    background: #ffd240;
+    @apply lil-w-full lil-h-48 lil-flex lil-items-center lil-rounded-lg;
+  }
+  .lil-reinforcement-title {
+    white-space: normal;
```
Comment (pritam248) [inline src/components/Reinforcement/reinforcement.css]: tailwind utility plz

File: src/components/Reinforcement/reinforcement.css [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+    @apply lil-flex-wrap lil-text-s-28 lil-text-left lil-font-bold;
+  }
+
+  .lil-reinforcement-button-wrapper {
+    @apply lil-flex lil-flex-wrap lil-mt-2 lil-justify-between;
+  }
+}
+
+.lil-reinforcement-button-wrapper {
```
Comment (pritam248) [inline src/components/Reinforcement/reinforcement.css]: plz move this above media query

File: src/components/Reinforcement/reinforcement.css [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.lil-reinforcement-title {
+  color: #4e3b00;
+  @apply lil-text-s-48 lil-text-left lil-font-bold;
+}
+
+@screen sm {
+  .lil-reinforcement {
+    background: #ffd240;
```
Comment (pritam248) [inline src/components/Reinforcement/reinforcement.css]: not needed

File: src/components/Reinforcement/reinforcement.css (hunk 9 above)
Comment (anshul-pinto0410) [reply src/components/Reinforcement/reinforcement.css:2]: I have added responsive design for small screens

File: src/components/Reinforcement/reinforcement.css [hunk 13]
```diff
@@ -0,0 +1,34 @@
+.lil-reinforcement {
+  height: 408px;
+  background-color: #ffd240;
+  background-image: radial-gradient(at top right, #1778f9 30%, #ffd240 10%);
```
Comment (pritam248) [inline src/components/Reinforcement/reinforcement.css:4]: 👍

File: src/components/Reinforcement/reinforcement.css (hunk 12 above)
Comment (anshul-pinto0410) [reply src/components/Reinforcement/reinforcement.css]: changed to background-image: none; as per suggestion

File: src/components/Card/FAQcard.css [hunk 14]
```diff
@@ -0,0 +1,22 @@
+.lil-faqcard {
+  /*
+  background-color needs to be updated
+  */
+  height: 280px;
+  width: 250px;
+  background-color: #1c314f;
+
+  @apply lil-rounded-xl lil-p-5  lil-cursor-pointer lil-text-center lil-m-2 lil-flex-col lil-justify-between;
```
Comment (pritam248) [inline src/components/Card/FAQcard.css:8]: nitpick: remove extra space

File: tailwind.config.js [hunk 15]
```diff
@@ -87,7 +88,17 @@ module.exports = {
...
+    opacity: {
+      '0': '0',
+      '20':'0.2',
+      '40':'0.4',
+      '25': '0.25',
+      '50': '0.5',
+      '60': '0.6',
+      '75': '0.75',
+      '80': '0.8',
+      '100': '1',
```
Comment (pritam248) [inline tailwind.config.js]: plz follow ascending order..... n let's follow gap of 25 for now...we can remove 40, 20, 60, 80

File: package.json [hunk 16]
```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "build": "npm-run-all -s build:tailwind build:uglify",
     "build:tailwind": "npx postcss src/littlegiant.css -o css/littlegiant.css",
+    "build:watch": "npx postcss src/littlegiant.css -o css/littlegiant.css --watch",
```
Comment (pritam248) [inline package.json]: nitpick: `build:tailwind-watch`

**Review by pritam248 — APPROVED** [review]

**Pull request merged**